### PR TITLE
Maint Drone Laws Tweaks

### PR DESCRIPTION
**Pull request by Geevies**
Ports https://github.com/Baystation12/Baystation12/pull/22386

1) You must repair, clean, and improve your assigned vessel, except where doing so would interfere with self-aware beings.
2) You must avoid interacting with self-aware beings, and may only interact with fellow maintenance drones.
3) You must not cause damage or harm to your assigned vessel or anything inside it.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -89,13 +89,13 @@
...
 /datum/ai_laws/drone
-	name = "Maintence Protocols"
+	name = "Maintenance Protocols"
 	law_header = "Maintenance Protocols"
 
 /datum/ai_laws/drone/New()
-	add_inherent_law("Preserve, repair and improve the station to the best of your abilities.")
-	add_inherent_law("Cause no harm to the station or crew.")
-	add_inherent_law("Interact with no humanoid or synthetic being that is not a fellow maintenance drone.")
+	add_inherent_law("You must repair, clean, and improve your assigned vessel, except where doing so would interfere with self-aware beings.")
```
Comment (alsoandanswer) [inline code/datums/ai_law_sets.dm]:
```suggestion
	add_inherent_law("You must repair, clean, and improve your assigned station, except where doing so would interfere with self-aware beings.")
```

File: code/datums/ai_law_sets.dm [hunk 2]
```diff
@@ -89,13 +89,13 @@
...
+	add_inherent_law("You must avoid interacting with self-aware beings, and may only interact with fellow maintenance drones.")
+	add_inherent_law("You must not cause damage or harm to your assigned vessel or anything inside it.")
```
Comment (alsoandanswer) [inline code/datums/ai_law_sets.dm]:
```suggestion
	add_inherent_law("You must not cause damage or harm to your assigned station or anything inside it.")
```

**Comment by skull132** [comment]
Hold for a tick, I'll consult the admins before pushing this.

**Comment by skull132** [comment]
Hold for a tick, I'll consult the admins before pushing this.

**Comment by Pratepresidenten** [comment]
Never been a fan of the improve bit of the law because no drone ever does this. They replace sturdy tables with inferior glass, paint on the floors and in one meme case, decided to lay reinforced flooring across the entire station because improvement.

Can this bit just be stripped?

**Comment by flimango** [comment]
Perhaps replace “improve” in law 1 with “maintain”?

**Comment by Cnaym** [comment]
Not a fan of the self aware part either, that has drones fighting carps, spiders, syndiborgs.
Something like "Do only interact with other drones or station infrastructure." would work better imho

**Comment by Geevies** [comment]
Admins should get together and form an opinion, various opinions will not be accepted.

**Pull request closed without merging**